README.md

```diff
@@ -8,10 +8,14 @@
 Trivial library for correct and stable conversions between `Task` and `Future`.  To use:
 
 ```sbt
-libraryDependencies += "io.verizon.delorean" %% "core" % "1.1.37"
+// For Scalaz 7.1.x
+libraryDependencies += "io.verizon.delorean" %% "core" % "1.1.37-scalaz-7.1"
```

Comment:
Here I mean that if you want Scalaz 7.1 take this string, if 7.2 take another one. I understand, it's may be confusing. I'll separate it to two different blocks.